playground/backend/internal/code_processing/code_processing.go

```diff
@@ -254,7 +304,7 @@ func finishByTimeout(ctx context.Context, pipelineId uuid.UUID, cacheService cac
 }
 
 // processError processes error received during processing code via setting a corresponding status and output to cache
-func processError(ctx context.Context, err error, data []byte, pipelineId uuid.UUID, cacheService cache.Cache, status pb.Status) {
+func processError(ctx context.Context, err error, data []byte, pipelineId uuid.UUID, cacheService cache.Cache, status pb.Status, stopReadLogsChannel, finishReadLogsChannel chan bool) {
```

Comment:
I agree with Ilya - too many arguments in a method. And every time getting more and more:)
Try to analyze the code and refactor somehow
Maybe pack channels to some structure (for example if `stopReadLogsChannel` and `finishReadLogsChannel` always used together) 
Maybe split responsibility of `processError` (and other such methods) to more methods

Reply:
Changed.